### PR TITLE
feature/D3ASIM-3695

### PR DESCRIPTION
**Pull request by ERafaelMartinez**
Integration test for testing the connection to the settlement market. This PR includes the setup file with the definition of the external-forecast assets.
This test also need the files from [this other PR](https://github.com/gridsingularity/gsy-e-sdk/pull/207) in the gsy-e-sdk to work.

### CODE REVIEW

File: src/gsy_e/setup/strategy_tests/external_devices_settlement_market.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+                          ),
+                    Asset("forecast-measurement-load",
+                          strategy=LoadForecastExternalStrategy(
+                              final_buying_rate=35, initial_buying_rate=11,
+                              use_market_maker_rate=True)
+                          ),
+
+                ], external_connection_available=True,
+            ),
+            Market("Cell Tower", strategy=LoadHoursStrategy(avg_power_W=100,
```
Comment (faizan2590) [inline src/gsy_e/setup/strategy_tests/external_devices_settlement_market.py:62]: Lets rename it to something generic like `Community Load`
```suggestion
            Market("Community Load", strategy=LoadHoursStrategy(avg_power_W=100,
```

**Review by hannesdiedrich — APPROVED** [review]
LGTM

File: src/gsy_e/setup/strategy_tests/external_devices_settlement_market.py (hunk 1 above)
Comment (ERafaelMartinez) [reply src/gsy_e/setup/strategy_tests/external_devices_settlement_market.py:62]: Sure, but could you please explain why you think this is needed?

**Review by hannesdiedrich — APPROVED** [review]
LGTM

**Review by faizan2590 — DISMISSED** [review]

**Review by hannesdiedrich — APPROVED** [review]

**Review by faizan2590 — APPROVED** [review]

**Pull request merged**